app/terminal/terminal.tsx

```diff
@@ -296,25 +296,8 @@ export default class TerminalComponent extends React.Component<TerminalProps, St
     this.updateLineLengthLimit();
   }
   private onDownloadClick() {
-    const serveLog = (log: string) => {
-      const element = document.createElement("a");
-      const plaintext = toPlainText(log);
-      element.setAttribute("href", "data:text/plain;charset=utf-8," + encodeURIComponent(plaintext));
-      element.setAttribute("download", "build_logs.txt");
-      element.click();
-    };
-    if (this.props.fullLogsFetcher) {
-      this.setState({ isLoadingFullLog: true });
-      this.props
-        .fullLogsFetcher()
-        .then(serveLog)
-        .catch((e) => errorService.handleError(e))
-        .finally(() => {
-          this.setState({ isLoadingFullLog: false });
-        });
-      return;
-    }
-    serveLog(this.props.value);
+    event.preventDefault();
```

Comment:
I think we should keep the default download behavior if `this.props.fullLogsFetcher` is undefined, otherwise this will throw an NPE. This is useful e.g. for downloading test logs (which uses the same Terminal component)

Reply:
sgtm
```suggestion
```